crates/cli-support/src/js/mod.rs

```diff
@@ -733,11 +753,11 @@ impl<'a> Context<'a> {
         // Initialize the `imports` object for all import definitions that we're
         // directed to wire up.
         let mut imports_init = String::new();
-        if self.wasm_import_definitions.len() > 0 {
-            imports_init.push_str("imports.");
-            imports_init.push_str(module_name);
-            imports_init.push_str(" = {};\n");
-        }
+
+        imports_init.push_str("imports.");
+        imports_init.push_str(module_name);
+        imports_init.push_str(" = {};\n");
```

Comment:
ℹ️ I have noticed that if a module has no imports and we don't initialize the `imports` with the `module_name` (`wbg`) that it will break when it tries to initialize the memory, because it gets assigned to `imports.wbg.memory`.